debugview/plugin.cpp

```diff
@@ -507,50 +434,9 @@ class debugview : public threadloop {
 
 		colorUniform = glGetUniformLocation(demoShaderProgram, "u_color");
 
-		groundObject.init(vertexPosAttr,
-			vertexNormalAttr,
-			colorUniform,
-			Ground_plane_NUM_TRIANGLES,
-			&(Ground_Plane_vertex_data[0]),
-			&(Ground_Plane_normal_data[0]),
-			GL_STATIC_DRAW
-		);
-
-		waterObject.init(vertexPosAttr,
-			vertexNormalAttr,
-			colorUniform,
-			Water_plane001_NUM_TRIANGLES,
-			&(Water_Plane001_vertex_data[0]),
-			&(Water_Plane001_normal_data[0]),
-			GL_STATIC_DRAW
-		);
-
-		treesObject.init(vertexPosAttr,
-			vertexNormalAttr,
-			colorUniform,
-			Trees_cone_NUM_TRIANGLES,
-			&(Trees_Cone_vertex_data[0]),
-			&(Trees_Cone_normal_data[0]),
-			GL_STATIC_DRAW
-		);
-
-		rocksObject.init(vertexPosAttr,
-			vertexNormalAttr,
-			colorUniform,
-			Rocks_plane002_NUM_TRIANGLES,
-			&(Rocks_Plane002_vertex_data[0]),
-			&(Rocks_Plane002_normal_data[0]),
-			GL_STATIC_DRAW
-		);
-
-		headsetObject.init(vertexPosAttr,
-			vertexNormalAttr,
-			colorUniform,
-			headset_NUM_TRIANGLES,
-			&(headset_vertex_data[0]),
-			&(headset_normal_data[0]),
-			GL_DYNAMIC_DRAW
-		);
+		// Load/initialize the demo scene.
+		demoscene = new ObjScene("demo2.obj");
```

Comment:
Let's make these values, not pointers, unless there is a compelling reason. Try removing the `new` and the `*` from 368. Lmk if this doesn't work for some reason.

Reply:
This doesn't work, because `ObjScene` does not have a default constructor, and C++ requires all members to have a default constructor... :/

`ObjScene` is constructed with the filename associated with the .obj file. This is obtained through reading envvars, which... can't be safely done with error checking in an initializer list.

Reply:
OK, I made a default constructor. All done.